apps/nextjs-website/src/components/atoms/GuideMenu/GuideMenu.tsx

```diff
@@ -149,17 +153,26 @@ const GuideMenu = ({
 }: GuideMenuProps) => {
   const { palette } = useTheme();
   const { shared } = translations;
-
+  const scrollUp = useScrollUp();
   const currentPath = usePathname();
   const segments = currentPath.split('/');
   const expanded = segments.map((_, i) => segments.slice(0, i + 1).join('/'));
+  const top = scrollUp
+    ? SITE_HEADER_HEIGHT + PRODUCT_HEADER_HEIGHT
+    : PRODUCT_HEADER_HEIGHT;
   return (
     <Box
       sx={{
         backgroundColor: palette.grey[50],
         display: 'flex',
         flexDirection: 'column',
-        position: 'relative',
+        flexShrink: 0,
+        position: 'sticky',
+        top: { lg: top },
+        height: 'calc(100vh - 100px)',
```

Comment:
Well done Jeremy, but I think we should remove the double scroll bar for small viewports:
[screencast-localhost_3000-2023.11.13-13_29_25.webm](https://github.com/pagopa/developer-portal/assets/39835990/51515299-5cc3-451e-ad2d-58abaff7158b)